File: examples/symmetry/src/email/client/email_client.c
#include <stddef.h>

typedef struct { const char *server; unsigned long sent_count; } email_client_t;
void email_client_connect(email_client_t *c, const char *server) { c->server = server; c->sent_count = 0; }
void email_client_send(email_client_t *c, const char *message) { (void)message; c->sent_count++; }

File: examples/symmetry/src/email/client/EmailClient.java
package symmetry.email.client;

public final class EmailClient {
    private String server;
    private long sentCount;
    public void connect(String server) { this.server = server; }
    public void send(String message) { sentCount++; }
    public String fetch() { return null; }
    public String getServer() { return server; }
    public long getSentCount() { return sentCount; }
}
